libraries/wbInterface/src/wbInterface.cpp

```diff
@@ -1229,6 +1232,34 @@ static void mdlStart (SimStruct* S) {
         return;
     }
 
+    switch (static_cast<int> (block_type)) {
+        case SET_PIDS:
+        {
+            //get
+            Value value;
+            value.fromString(additionalParameter.c_str());
```

Comment:
Because I'm always confused by Bottle, properties and all these stuff....
I just want to parse a list from a string :disappointed: